### PR TITLE
@damassi => Exclude unused fields from news admin

### PR DESCRIPTION
**Pull request by eessex**
Removes fields from Admin panel that are not used in News articles. 
Related to [GROW-433](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-433)

### CODE REVIEW

File: src/client/apps/edit/components/admin/components/article/article_authors.jsx [hunk 1]
```diff
@@ -101,14 +101,15 @@ export class ArticleAuthors extends Component {
...
-          <div className='field-group'>
-            <AutocompleteListMetaphysics
-              field='contributing_authors'
-              label='Contributing Authors'
-              model='users'
-            />
-          </div>
+          {article.layout !== 'news' &&
+            <div className='field-group'>
+              <AutocompleteListMetaphysics
```
Comment (damassi) [inline src/client/apps/edit/components/admin/components/article/article_authors.jsx]: Really nice work on this pattern BTW 👍

File: src/client/apps/edit/components/admin/index.jsx [hunk 2]
```diff
@@ -55,12 +66,13 @@ export class EditAdmin extends Component {
...
+          {!isNews &&
+            <AdminAppearances />
+          }
+          {isEditorial && !isNews &&
             <AdminSuperArticle />
           }
-          {isEditorial &&
+          {isEditorial && !isNews &&
             <AdminSponsor />
           }
```
Comment (damassi) [inline src/client/apps/edit/components/admin/index.jsx]: Can these cases be collapsed together?

```javascript 
{isEditorial && !isNews &&
  <Fragment>
    <AdminSuperArticle />
    <AdminSponsor />
  </Fragment>   
}
```

File: src/client/apps/edit/components/admin/index.jsx [hunk 3]
```diff
@@ -70,7 +82,8 @@ export class EditAdmin extends Component {
 }
 
 const mapStateToProps = (state) => ({
-  channel: state.app.channel
+  isNews: state.edit.article.layout === 'news',
+  isEditorial: state.app.channel.type === 'editorial'
```
Comment (damassi) [inline src/client/apps/edit/components/admin/index.jsx]: 👍

File: src/client/apps/edit/components/admin/test/components/article/article_authors.test.js [hunk 4]
```diff
@@ -83,6 +83,12 @@ describe('ArticleAuthors', () => {
     expect(component.find(AutocompleteListMetaphysics).length).toBe(1)
   })
 
+  it('Does not render contributing authors if layout is news', () => {
+    props.article.layout = 'news'
+    const component = getWrapper(props)
+    expect(component.find(AutocompleteListMetaphysics).length).toBeFalsy()
```
Comment (damassi) [inline src/client/apps/edit/components/admin/test/components/article/article_authors.test.js]: One totally unrelated thought that I had from your presentation today is that the `<AutocompleteListMetaphysics />` component could possibly be collapsed into just `<AutocompleteList />` with a datasource prop:

```javascript
<AutocompleteList
  dataSource={{
    metaphysics: {
      field: 'contributing_authors',
      label: 'Contributing Authors',
      model: 'foo'
    }
  }}
/>
```

File: src/client/apps/edit/components/admin/index.jsx (hunk 2 above)
Comment (damassi) [reply src/client/apps/edit/components/admin/index.jsx]: That should work, but not too sure; might want to investigate since fragments are new.

Comment (eessex) [reply src/client/apps/edit/components/admin/index.jsx]: Gonna merge without and keep this in mind for the future.

**Pull request merged**